### ISSUE TITLE
Add New Custom Notification Watching

### ISSUE THREAD

**Issue by landry314** [slapperwan/gh4a#1053]
Currently it appears you can only watch or unwatch a repository. I want to watch a repository just for releases. Now github has been updated to add custom notifications to watch for "Issues", "Pull Requests", or "Releases". It would be nice to be able to manipulate this directly in Octodroid.

**Comment by iamtalhaasghar** [comment]
Exactly, there should be some option to subscribe to custom notifications for a repo at least for "releases". We can trigger notifications for "issues" and "pull requests" by participating but there is no way in Octodroid to do so for releases.

**Comment by yephny** [comment]
Does Github API allow this?
